addons/mail/static/src/discuss/core/common/action_panel.xml

```diff
@@ -2,15 +2,22 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ActionPanel">
-        <div class="o-mail-ActionPanel overflow-auto d-flex flex-column flex-shrink-0 position-relative py-3 pt-0 h-100" t-att-class="{ 'px-3 bg-view': !env.inChatter }">
-            <div class="o-mail-ActionPanel-header position-sticky top-0 py-3 d-flex align-items-baseline" t-att-class="{ 'bg-view': !env.inChatter }">
-                <button t-if="env.closeActionPanel" class="o-mail-ActionPanel-backButton btn opacity-75 opacity-100-hover ps-0 py-0 fs-5" title="Close panel" t-on-click.stop="env.closeActionPanel">
-                    <i class="oi oi-arrow-left"/>
-                </button>
-                <p t-if="props.title" class="fs-6 fw-bold text-uppercase m-0 text-700 flex-grow-1" t-esc="props.title"/>
-            </div>
-            <t t-slot="default"/>
+        <ResizablePanel t-if="props.resizable and !env.inChatWindow" class="classNames" handleSide="'start'">
+            <t t-call="mail.ActionPanelContent"/>
+        </ResizablePanel>
+        <div t-else="" t-att-class="classNames">
+            <t t-call="mail.ActionPanelContent"/>
         </div>
     </t>
 
+    <t t-name="mail.ActionPanelContent">
```

Comment:
`mail.ActionPanelContent` => `mail.ActionPanel.content`. Follow BEM naming convention so seeing the template name gives clear traceability of component name and part name)